framework/components/ema-list-items/ema-list-items.twig

```diff
@@ -1 +1,41 @@
 {% include '@ec-europa/ecl-list-items' %}
```

Comment:
We should remove the border between the list item and the translations, and add a border below translations. So that it is more obvious that translations are related to this item